docs/tutorials/state_histograms.ipynb

```diff
@@ -0,0 +1,293 @@
+{
```

Comment:
<p>Question: what is the <code>plt.subplots()[1]</code>  doing here? Could we just use the default axes instead?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/4024/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
It will create a new subplot and pass the axes to the function. If we don't pass a new axes, the `plot_state_histogram` will create a new one itself but then it'll also implicitly call `fig.show()` which leads to an ugly warning regarding the back-end being a non-GUI backend. 

Tl;Dr : This is needed to ensure there are no unnecessary warnings in the notebook.